piqueserver/game_modes/infiltration.py

```diff
@@ -78,8 +78,9 @@ def score(self):
         intel_capture.winning = winning
         self.protocol.send_contained(intel_capture, save=True)
         if winning:
-            self.team.initialize()
-            self.team.other.initialize()
+            self.team = self.team(*self.team.get_init_values())
```

Comment:
Couldn't this just call `self.reset_teams()`?

Reply:
Sorry could be wrong; I'm not that familiar with the infiltration code.

Reply:
The infiltration code is a bit weird. This piece is from the DummyPlayer class which isn't a protocol so there is no way to use reset_teams from the ServerProtocol. Ideally resetting should be handled by the InfiltrationProtocol but that would probably require some pretty big changes in the infiltration code.